x-pack/plugins/security/public/views/management/roles_grid/components/roles_grid_page.tsx

```diff
@@ -0,0 +1,312 @@
+                sort: {
+                  field: 'name',
+                  direction: 'asc',
+                },
+              }}
+              // @ts-ignore missing rowProps typedef
+              rowProps={() => {
+                return {
+                  'data-test-subj': 'roleRow',
+                };
+              }}
+              isSelectable
+            />
+          }
+        </EuiPageContentBody>
+      </EuiPageContent>
+    );
+  };
+
+  private getColumnConfig = (intl: InjectedIntl) => {
+    const reservedRoleDesc = intl.formatMessage({
+      id: 'xpack.security.management.roles.reservedColumnDescription',
+      defaultMessage: 'Reserved roles are built-in and cannot be edited or removed.',
+    });
+
+    return [
+      {
+        field: 'name',
+        name: intl.formatMessage({
+          id: 'xpack.security.management.roles.nameColumnName',
+          defaultMessage: 'Role',
+        }),
+        sortable: true,
+        truncateText: true,
+        render: (name: string, record: Role) => {
+          return (
+            <span>
+              <EuiLink data-test-subj="roleRowName" href={getRoleManagementHref(name)}>
+                {name}
+              </EuiLink>
+              {!isRoleEnabled(record) && (
+                <EuiIconTip
+                  type="alert"
+                  size="xs"
+                  color="warning"
+                  data-test-subj="disabledRoleTip"
+                  title={
+                    <FormattedMessage
+                      id="xpack.security.management.roles.disabledTooltip"
+                      defaultMessage="This role is currently disabled. You may only view or delete it."
+                    />
+                  }
+                />
+              )}
+            </span>
+          );
+        },
+      },
+      {
+        field: 'metadata._reserved',
+        name: (
+          <div>
+            <FormattedMessage
+              id="xpack.security.management.roles.reservedColumnName"
+              defaultMessage="Reserved"
+            />
+            <span className="rolesGridPage__reservedRoleTooltip">
+              <EuiIconTip
+                type="questionInCircle"
+                size="m"
+                color="euiColorMediumShade"
+                className="rolesGridPage__reservedRoleTooltip"
+                title={reservedRoleDesc}
+              />
+            </span>
+          </div>
+        ),
+        sortable: true,
+        dataType: 'boolean',
+        align: 'right',
+        description: reservedRoleDesc,
+        render: (reserved: boolean | undefined) => {
+          const label = intl.formatMessage({
+            id: 'xpack.security.management.roles.reservedRoleIconLabel',
+            defaultMessage: 'Reserved role',
+          });
+
+          return reserved ? (
+            // @ts-ignore missing "title" proptype
+            <EuiIcon aria-label={label} title={label} data-test-subj="reservedRole" type="check" />
```

Comment:
`title` is not a supported tag on `svg` (which is what `EuiIcon` resolves to) and that's why you're getting a TS error. Wrap the SVG in a span an apply the title attribute to that.